### PR TITLE
PS-135: fix (JobFactory): init new job with empty result

### PR DESCRIPTION
**Pull request by MiroCillik**
Depends on https://github.com/keboola/job-queue-internal-api/pull/47. Tests are failing.

### CODE REVIEW

**Comment by odinuv** [comment]
a nestaci jenom zmenit tohle 
https://github.com/keboola/job-queue-internal-api-php-client/blob/b25d1f2645ce0caeebb8b0e6fc670768d2e9e93a/src/JobFactory/Job.php#L62 aby to vracelo vzdycky pole ?

**Comment by MiroCillik** [comment]
To som si nevsimol, ze je to tam takto :) To by asi stacilo.

**Comment by MiroCillik** [comment]
Ale nestaci len to, musi to byt v tom poli $data. Lebo ked ten job serializujes do JSONu, tak nevolas tieto get* metody ale pouzijes rovno to pole $data v Jobe.

**Comment by odinuv** [comment]
jasne, mas pravdu! jen teda ten getter uz nebude nikdy vracet null

File: docker-compose.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ services:
       - ./:/code
 
   internal-api:
-    image: keboola/docker-job-queue-internal-api
```
Comment (odinuv) [inline docker-compose.yml]: prihod tam docker pull do .yml

Comment (MiroCillik) [reply docker-compose.yml]: jo, akurat to riesim, pozeral som ci do docker-compose nepridali nejaky parameter, ktorym by sa to dalo forcnut, vela tomu nechyba, ale este to tam nie je :)

Comment (MiroCillik) [reply docker-compose.yml]: https://github.com/docker/compose/issues/3574

File: src/JobFactory/Job.php [hunk 2]
```diff
@@ -59,7 +59,7 @@ public function getProjectId(): string
 
     public function getResult(): ?array
```
Comment (odinuv) [inline src/JobFactory/Job.php]: array

**Pull request merged**